vuepress/docs/next/tutorials/devops/local-tips-and-tricks.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebarDepth: 2
+--- 
+# Local Development Tips and Tricks
+Here you'll find a list of tips and tricks for local development environments. Please feel free to 
+provide your own or to discuss others on the [forum](https://forum.entando.org). 
+
+## Kubernetes
+Per [Getting Started](../../docs/getting-started/) we've recommended using Multipass as
+a quick way to spin up an Ubuntu VM and host a local Kubernetes cluster. Practically speaking most
+teams utilize a shared Kubernetes cluster, managed by an operations team and installed either on premise
+or your cloud provider of choice, for full integration testing, CI/CD, DevOps, etc. but there are many times
+when a local environment is useful. 
+
+## Windows development
+### Hyper-V IP changes
+**Q:** By design, Windows Hyper-V changes the IP address assigned to a VM (Multipass or other) when Windows 
+is restarted. How do I deal with this?
+
```

Comment:
Entando stops working when I restart my VM on Windows. How can I fix this?

Reply:
I'll tweak it but it's not correct that it stops working when the VM is restarted. It only stops working when WIndows is restarted.